rllib/algorithms/algorithm.py

```diff
@@ -198,6 +198,13 @@ class Algorithm(Trainable):
     # List of keys that are always fully overridden if present in any dict or sub-dict
     _override_all_key_list = ["off_policy_estimation_methods"]
 
+    _progress_metrics = [
```

Comment:
since this is referenced outside the module, could we make it public?

Reply:
I'd like to keep it internal for now as we will want to refactor all of progress reporting soon (see also https://github.com/ray-project/ray/issues/24604)

I don't want to rush out a public seeming API before we design that. Shall we keep this internal for now and consider promoting once we tackle the reporting?